From a74c04a64c1d2524beab59db948ad6641f63af8b Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Tue, 3 May 2016 10:02:30 +0300
Subject: [PATCH 1078/1240] net: mvpp2x: register axi config moved to
 mv_pp2x_axi_config

- register axi config moved to mv_pp2x_axi_config

Change-Id: Iadfe0725a2479a8aa06cccb2dc65d08dc43d9fc9
Reviewed-on: http://vgitil04.il.marvell.com:8080/29435
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
---
 drivers/net/mv_pp2x.c | 39 ++++++++++++++++++++-------------------
 1 file changed, 20 insertions(+), 19 deletions(-)

diff --git a/drivers/net/mv_pp2x.c b/drivers/net/mv_pp2x.c
index e94ea1b..2c6d6ae 100644
--- a/drivers/net/mv_pp2x.c
+++ b/drivers/net/mv_pp2x.c
@@ -2904,23 +2904,6 @@ static int mv_pp2x_probe(struct eth_device *dev)
 		return err;
 	}
 
-	mv_pp2x_write(pp->pp2, MVPP22_AXI_BM_WR_ATTR_REG,
-				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
-	mv_pp2x_write(pp->pp2, MVPP22_AXI_BM_RD_ATTR_REG,
-				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
-	mv_pp2x_write(pp->pp2, MVPP22_AXI_AGGRQ_DESCR_RD_ATTR_REG,
-				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
-	mv_pp2x_write(pp->pp2, MVPP22_AXI_TXQ_DESCR_WR_ATTR_REG,
-				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
-	mv_pp2x_write(pp->pp2, MVPP22_AXI_TXQ_DESCR_RD_ATTR_REG,
-				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
-	mv_pp2x_write(pp->pp2, MVPP22_AXI_RXQ_DESCR_WR_ATTR_REG,
-				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
-	mv_pp2x_write(pp->pp2, MVPP22_AXI_RX_DATA_WR_ATTR_REG,
-				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
-	mv_pp2x_write(pp->pp2, MVPP22_AXI_TX_DATA_RD_ATTR_REG,
-				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
-
 	/* Enable HW PHY polling */
 #if defined(CONFIG_MVPPV21)
 	u32 val;
@@ -3570,10 +3553,28 @@ static void mv_pp2x_axi_config(struct mv_pp2x *pp2)
 	/* Config AXI Read&Write Normal and Soop mode  */
 	mv_pp2x_write(pp2, MVPP22_AXI_RD_NORMAL_CODE_REG,
 				MVPP22_AXI_RD_CODE_MASK);
-	mv_pp2x_write(pp2, MVPP22_AXI_RD_SNP_CODE_REG, MVPP22_AXI_RD_CODE_MASK);
+	mv_pp2x_write(pp2, MVPP22_AXI_RD_SNP_CODE_REG,
+				MVPP22_AXI_RD_CODE_MASK);
 	mv_pp2x_write(pp2, MVPP22_AXI_WR_NORMAL_CODE_REG,
 				MVPP22_AXI_WR_CODE_MASK);
-	mv_pp2x_write(pp2, MVPP22_AXI_WR_SNP_CODE_REG, MVPP22_AXI_WR_CODE_MASK);
+	mv_pp2x_write(pp2, MVPP22_AXI_WR_SNP_CODE_REG,
+				MVPP22_AXI_WR_CODE_MASK);
+	mv_pp2x_write(pp2, MVPP22_AXI_BM_WR_ATTR_REG,
+				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
+	mv_pp2x_write(pp2, MVPP22_AXI_BM_RD_ATTR_REG,
+				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
+	mv_pp2x_write(pp2, MVPP22_AXI_AGGRQ_DESCR_RD_ATTR_REG,
+				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
+	mv_pp2x_write(pp2, MVPP22_AXI_TXQ_DESCR_WR_ATTR_REG,
+				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
+	mv_pp2x_write(pp2, MVPP22_AXI_TXQ_DESCR_RD_ATTR_REG,
+				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
+	mv_pp2x_write(pp2, MVPP22_AXI_RXQ_DESCR_WR_ATTR_REG,
+				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
+	mv_pp2x_write(pp2, MVPP22_AXI_RX_DATA_WR_ATTR_REG,
+				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
+	mv_pp2x_write(pp2, MVPP22_AXI_TX_DATA_RD_ATTR_REG,
+				MVPP22_AXI_ATTR_SNOOP_CNTRL_BIT);
 }
 
 DECLARE_GLOBAL_DATA_PTR;
-- 
1.9.1

